aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,134 @@
+
+## Installation
+
+Follow the guidance in [Installation (.NET MAUI documentation)](/dotnet/maui/get-started/installation).
+
+## Create a .NET MAUI Blazor app
+
+Launch Visual Studio 2022 17.1 Preview, and in the start window click **Create a new project** to create a new project:
+
+:::image type="content" source="maui/_static/new-solution.png" alt-text="New solution.":::
+
+In the **Create a new project** window, select **MAUI** in the **Project type** drop-down, select the **.NET MAUI App (Preview)** template, and click the **Next** button:
+
+:::image type="content" source="maui/_static/new-project.png" alt-text="Choose a template.":::
+
+> [!NOTE]
+> If the **.NET MAUI App (Preview)** project template does not appear in the **Create a new project** window, open the Developer Command Prompt for VS 2022 Preview and run the following command: `devenv.exe /updateConfiguration`. Then, restart Visual Studio.
+
+In the **Configure your new project** window, name your project, choose a suitable location for it, and click the **Create** button:
+
+:::image type="content" source="maui/_static/configure-project.png" alt-text="Configure the project.":::
+
+Wait for Visual Studio to create the project and for the project's dependencies to be restored:
+
+:::image type="content" source="maui/_static/restored-dependencies.png" alt-text="Restored dependencies.":::
+
+In the **Android SDK License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/android-sdk-license.png" alt-text="Android SDK License Acceptance window.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+Wait for Visual Studio to download the Android SDK and Android Emulator.
+
+## Run the app in the Windows Machine Emulator
+
+In the Visual Studio toolbar, select the **Windows Machine** button to build the app:
+
+:::image type="content" source="maui/_static/windows-machine-button.png" alt-text="Windows Machine button.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+If Developer Mode isn't enabled, you're prompted to enable it in **Settings** > **For developers** > **Developer Mode**. Set the switch to **On**:
+
+:::image type="content" source="maui/_static/windows-developer-mode.png" alt-text="Windows Developer Mode enabled.":::
+
+The running app in the Windows Machine Emulator:
+
+:::image type="content" source="maui/_static/running-app-windows-1.png" alt-text="App running in the Windows Machine Emulator.":::
```

Comment:
Can we show this screen shot with the Windows 11 styling? Not a big deal if that's a pain to get.

Reply:
I was only pointing out that the screen shots look a bit different on my Win11 machine (rounded corners, etc):

![image](https://user-images.githubusercontent.com/1874516/153786668-ccf2ee69-4b52-4074-8417-5346a98ade5e.png)

Reply:
I see. Ok ... I'll check on this. It might just be that the image was saved without transparency.

Reply:
I'm on Win10, so perhaps that explains it.

I'll use the shot that you just sent ... I'll Photoshop the app name on it.